src/redux/news/news.reducer.js

```diff
@@ -3,7 +3,7 @@ import { SET_NEWS, SET_NEWS_ARTICLE, SET_LOADING } from './news.types';
 const initialState = {
   loading: true,
   list: [],
-  item: {}
+  item: []
```

Comment:
Please, initiate value of `item` with `null` and check in component `loading || !item`

Reply:
And, please, rename it to something like `activeArticle` because it is much more explicit

Reply:
Done